services/tests/e2e/docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
             },
             "http://localhost:8070": {
               "authority": "http://authority-without-discrovery-endpoint",
-              "audience": "Stitch Gateway",
+              "audience": ["Stitch Gateway", "stitch gateway"],
```

Comment:
looks like now you are only testing multiple audience but not single audience

Reply:
There is another issues above on port 8060..